src/Features/Core/Portable/Completion/CompletionService.cs

```diff
@@ -197,13 +207,25 @@ public virtual TextSpan GetDefaultCompletionListSpan(SourceText text, int caretP
         /// This character may be used as part of the change. 
         /// This value is null when the commit was caused by the [TAB] or [ENTER] keys.</param>
         /// <param name="cancellationToken"></param>
-        public virtual Task<CompletionChange> GetChangeAsync(
+        public virtual async Task<CompletionChange> GetChangeAsync(
             Document document,
             CompletionItem item,
             char? commitCharacter = null,
             CancellationToken cancellationToken = default)
         {
-            return Task.FromResult(CompletionChange.Create(new TextChange(item.Span, item.DisplayText)));
+            var provider = GetProvider(item);
+            if (provider != null)
```

Comment:
can this now ever be null?

Reply:
Yes, see comment here
https://github.com/dotnet/roslyn/pull/61943#discussion_r899395628